templates/base.lds

```diff
@@ -87,11 +87,11 @@ SECTIONS
      * certain core features */
     PROVIDE(__metal_chicken_bit = {{ chicken_bit|default(0) }});
 
-{% if eccscrub_en %}
+{#{% if eccscrub_en %}#}
```

Comment:
if this is no longer need and already verify that it I the case, better to simply delete this,